liby2/src/Y2SerialComponent.cc

```diff
@@ -344,23 +344,25 @@ bool Y2SerialComponent::initializeConnection()
 
 	 // first write one space so the other side gets
 	 // fullfilled its necessities
-	 write(fd_serial, space_buf, 1);
+	 (void) (write(fd_serial, space_buf, 1) == 1);
```

Comment:
I doubt anyone knows, because this is the *serial line component* which is IMHO long unused. My intent is to fix the warnings without much effort, and drop this for SLE13.